gensrc/thrift/FrontendService.thrift

```diff
@@ -1735,6 +1735,22 @@ struct TReportLakeCompactionResponse {
     1: optional bool valid
 }
 
+struct TListSessionOptions {
+    1: optional bool temporary_table_only;
+}
+
+struct TListSessionRequest {
+    1: optional TListSessionOptions options;
+}
+
+struct TSessionInfo {
+    1: optional string session_id;
+}
+
+struct TListSessionResponse {
+    1: optional list<TSessionInfo> sessions;
```

Comment:
Add `1: optional Status.TStatus status` to tell the client if this operation success or not.